NhkEasyPracticeWithWaniKani.user.js

```diff
@@ -62,6 +67,26 @@ function findRuby(knownVocab) {
   });
 }
 
+/**
+ * Finds the Ruby tags in NHK Easy's pop-up dictionary and replaces them with 
+ * just the word if the word is in the known set of vocabulary.
+ * @param {Set} knownVocab Set of all vocab that has been learned on WK
+ */
+function findRubyInDictionary(knownVocab) {
```

Comment:
Do you have a way to just set the ruby to hidden instead of replacing it? My goal, although I don't know when I'll get to it, was to make it so I can click on words and show the ruby later. This would be good for if I forget a word, or the app hides the ruby incorrectly.

This is how it works on bunpro, anther web app I use for grammar and I was wanting to mimic this behavior as I found it really useful.